OloEngine/src/OloEngine/Renderer/Debug/FrameCaptureManager.cpp

```
@@ -0,0 +1,228 @@
+        m_PendingFrame.PostBatchCommands.clear();
+        DeepCopyCommands(bucket, m_PendingFrame.PostBatchCommands, true);
+        m_HasPendingPostBatch = true;
+    }
+
+    void FrameCaptureManager::OnFrameEnd(u32 frameNumber, f64 sortTimeMs, f64 batchTimeMs, f64 executeTimeMs)
+    {
+        if (!IsCapturing())
+            return;
+
+        m_PendingFrame.FrameNumber = frameNumber;
+        m_PendingFrame.TimestampSeconds =
```

Cppcheck (2.19.0)

[style] The function 'GetGroupID' is never used.

(unusedFunction)